docs_2.0/manual_EN/3.quick-start/2.deploy-nebula-graph-with-docker-compose.md

```diff
@@ -0,0 +1,184 @@
+  | Docker Compose | Latest              | [Install Docker Compose](https://docs.docker.com/compose/install/) |
+  | Git            | Latest              | [Download Git for Linux](https://git-scm.com/download/linux)       |
+
+* If you are deploying Nebula Graph as a non-root user, grant the user with Docker-related privileges. For a detailed instruction, see [Docker docs: Manage Docker as a non-root user](https://docs.docker.com/engine/install/linux-postinstall/#manage-docker-as-a-non-root-user).
+
+* You have started the Docker service on your host.
+
+* If you have already deployed another version of Nebula Graph with Docker Compose on your host, to avoid compatibility issues，back up [the service data](#check-the-service-data-and-logs) if you need, and delete the `nebula-docker-compose/data` directory.
+
+## How to deploy
+
+1. Clone the `v2-preview` branch of the `nebula-docker-compose` repository to your host with Git.
+
+[TODO: Update the branch name on the release on 2.0 alfa.]
+
+> **DON'T**: The `v2-preview` branch stores the Docker Compose solution for the latest Nebula Graph development release, the 2.0 pre-release. DON'T use this release for production.
+
+    ```bash
+    $ git clone --branch v2-preview https://github.com/vesoft-inc/nebula-docker-compose.git
+    ```
+
+1. Go to the `nebula-docker-compose` directory.
+
+    ```bash
+    $ cd nebula-docker-compose/
+    ```
+
+2. Run the following command to start all the Nebula Graph services.
+
+    ```bash
+    $ docker-compose up -d
+    ```
+
+    The following information indicates the services have started:
+
+    ```bash
+    Creating nebula-docker-compose_metad2_1 ... done
+    Creating nebula-docker-compose_metad1_1 ... done
+    Creating nebula-docker-compose_metad0_1 ... done
+    Creating nebula-docker-compose_storaged2_1 ... done
+    Creating nebula-docker-compose_graphd_1    ... done
+    Creating nebula-docker-compose_storaged0_1 ... done
+    Creating nebula-docker-compose_storaged1_1 ... done
+    ```
+
+    >**NOTE**: For more information of the preceding services, see [Design and Architecture of Nebula Graph](https://docs.nebula-graph.io/manual-EN/1.overview/3.design-and-architecture/1.design-and-architecture/).
+
+3. Use nebula-console to connect to Nebula Graph.
+
+    [Nebula-console](https://github.com/vesoft-inc/nebula-console) is the native CLI client of Nebula Graph. In this step, Docker pulls the nebula-console images automatically from Docker Hub according to the path we set in the following commands and uses it to connect to the graphd service of Nebula Graph. You can use other clients to connect to Nebula Graph instead of Nebula-console, such as [Nebula Graph Studio](https://github.com/vesoft-inc/nebula-web-docker) and [clients for different programming languages](https://docs.nebula-graph.io/manual-EN/1.overview/2.quick-start/3.supported-clients/).
```

Comment:
to the path we set in the following commands ==> to the path you set in the following commands

always avoid using "we"